README.md

```diff
@@ -169,10 +169,13 @@ bert-base-uncased/
 
 ## Supported tasks/models
 
-Here is the list of all tasks and models currently supported by Transformers.js.
+Here is the list of all tasks and architectures currently supported by Transformers.js.
 If you don't see your task/model listed here or it is not yet supported, feel free
 to open up a feature request [here](https://github.com/xenova/transformers.js/issues/new/choose).
 
+To find compatible models on the Hub, select the "transformers.js" library tag in the filter menu (or visit [this link](https://huggingface.co/models?library=transformers.js)).
+You can refine your search by selecting the task you're interested in (e.g., [text-classification](https://huggingface.co/models?pipeline_tag=text-classification&library=transformers.js)).
```

Comment:
i see!

Then I'm wondering if there's a way to tag the repos that contain `Optimum-exported ONNX files`, like for instance my repo here where i add the converted files directly to the source repo: https://huggingface.co/julien-c/EsperBERTo-small-pos/tree/main

(BTW i would assume many (most?) of the ONNX files on the Hub were generated through Optimum)

Maybe we could base ourselves on the presence of a `onnx/model_quantized.onnx` file in the repo, or something similar

Reply:
> like for instance my repo here where i add the converted files directly to the source repo: https://huggingface.co/julien-c/EsperBERTo-small-pos/tree/main

Ahh yes, good point. One option is that the user could just simply add "transformers.js" (or "optimum") as another tag, and it should still show up when you filter by "transformers.js". I duplicated your model and just added another tag:
![image](https://github.com/xenova/transformers.js/assets/26504141/d9166937-fbf6-474c-a212-8226d9de5370)

and indeed, it does filter correctly:
![image](https://github.com/xenova/transformers.js/assets/26504141/c9dc5014-baaf-4e04-8bcf-33ec82352135)

> (BTW i would assume many (most?) of the ONNX files on the Hub were generated through Optimum)

I'd be interested to know myself 👀 However, the majority of the most popular models (e.g., https://huggingface.co/xlm-roberta-large) did not use optimum.

> Maybe we could base ourselves on the presence of a onnx/model_quantized.onnx file in the repo, or something similar

Unfortunately, not so simple 😅 it depends on the type of model:
- encoder-only: "model.onnx"
- encoder-decoder: "encoder_model.onnx" + "decoder_model_merged.onnx" 
- decoder-only: "decoder_model_merged.onnx"

(and the `_quantized` can be appended to any one)


... and even in cases where models are exported by optimum, they need to make sure the correct settings are used. For example, https://huggingface.co/optimum/t5-small was exported with optimum, but doesn't have the merged decoder:
![image](https://github.com/xenova/transformers.js/assets/26504141/e0a200cc-2660-40c4-a0f9-b65e4540c030). This is because the merged behaviour was only added ~3 months ago (https://github.com/huggingface/optimum/commit/0c3713d88be934d880203a126374139e3c0e19b2) while that export is 5 months old.

Reply:
> However, the majority of the most popular models (e.g., https://huggingface.co/xlm-roberta-large) did not use optimum.

I beg to differ: https://huggingface.co/xlm-roberta-large/discussions/11 😄

But yeah, i see your point, it was an old version of optimum export.

Okay, let's keep this question in the back of our head for now then! 

Thanks for the detailed replies!!

Reply:
Whoops sorry! Linked wrong one haha - I searched a few hours ago and found one, which had a bunch of small files. let me get it quickly.

Reply:
Ahh here it is: https://huggingface.co/pszemraj/flan-t5-large-grammar-synthesis/tree/main/onnx

Actually, looking back at the top ones (any task), most of them are actually using exports from optimum. I think I had the text2text filter on.

Reply:
>Okay, let's keep this question in the back of our head for now then!

Agreed! Definitely something to consider as WebML grows. For now, I think it's okay to require users to export via the latest versions of optimum (or just use one of the ones we already provide).

> Thanks for the detailed replies!!

🤗 You're welcome!